### ISSUE TITLE
name of merged grid

### ISSUE THREAD

**Issue by tischi** [mobie/mobie-viewer-fiji#814]
Hi,

Is it important that the `name` of a `MergedGridTransformer`
is `protected String mergedGridSourceName` and not just `protected String name` as for the `TransformedGridTransformer`?

**Comment by KateMoreva** [comment]
> Hi,
> 
> Is it important that the `name` of a `MergedGridTransformer` is `protected String mergedGridSourceName` and not just `protected String name` as for the `TransformedGridTransformer`?

We also has a field name:String in the base class org.embl.mobie.viewer.transform.AbstractSourceTransformer

**Comment by constantinpape** [comment]
The reason we chose to add `mergedGridSourceName` is that merged grid actually creates a new source - and this field will give the name of the source, whereas the other transformations just have the `name` field to allow identification, but they keep the number of sources unchanged, and the names of sources are determined by `sourceNamesAfterTransform`.

**Comment by tischi** [comment]
Good point! Thanks!

**Comment by constantinpape** [comment]
> More consistent may be: `sourceNameAfterTransform`?!

Yeah, would be more consistent. (But it doesn't really make sense to update the spec for pure name changes - this is quite a lot of work for me. We could make a list of potential name changes though and implement all of them at some point.)

**Comment by tischi** [comment]
`sourceDisplays` => `displays` ?!
